src/pool.rs

```diff
@@ -9,8 +9,8 @@ use crate::Proxy;
 
 use url::Url;
 
-const DEFAULT_MAX_IDLE_CONNECTIONS: usize = 100;
-const DEFAULT_MAX_IDLE_CONNECTIONS_PER_HOST: usize = 1;
+pub const DEFAULT_MAX_IDLE_CONNECTIONS: usize = 100;
```

Comment:
Let's actually move these values over into `agent.rs`, since notionally it's the Agent that provides the defaults now.

Reply:
Ok!